python/openassetio/managerAPI/ManagerInterface.py

```diff
@@ -998,27 +1047,42 @@ def setRelatedReferences(
         """
         Creates a new relationship between the referenced entities.
 
-        @param append bool, When True (default) new relationships will
-        be added to any existing ones. If False, then any existing
-        relationships with the supplied specification will first be
-        removed.
-
         Though getRelatedReferences is an essential call, there is some
         asymmetry here, as it is not necessarily required to be able to
         setRelatedReferences directly. For example, in the case of a
         'shot' (as illustrated in the docs for getRelatedReferences) -
         any new shots would be created by registering a new
         ShotSpecification under the parent, rather than using this call.
         The best way to think of it is that this call is reserved for
-        defining relationships between existing assets (Such as
-        connecting multiple image sequences published under the same
-        shot, as being part of the same render.) and 'register' as being
-        defining the relationship between a new asset and some existing
-        one.
+        defining relationships between existing assets (such as
+        connecting the script used to define a render, with the image
+        sequences it creates) and 'register' as being defining the
+        relationship between a new asset and some existing one.
 
         In systems that don't support post-creation adjustment of
         relationships, this can simply be a no-op.
 
+        @param entityRef str, The entity to which the relationship
```

Comment:
[consider] Would be good to be consistent with the `@param` style, at least within a given file unless/until we decide on a global style
```suggestion
        @param entityRef `str` The entity to which the relationship
```